wordcounter.rb

```diff
@@ -0,0 +1,25 @@
+class Counter
+
+	def countWords(text)
+	  text = text.downcase.gsub(/[^a-z' ]/, '') #make lowercase and only keep allowed characters
+    raw_words = text.split(" ")
+    count = {}
```

Comment:
anytime you initialize a new object just to build it up later is a good time to think about using `.map` or `.reduce.`

in this case `.reduce` or `.each_with_object` is the way you want to go here. These would also simplify your code greatly.